### DISCUSSION TITLE
Test received http 200 code but shows 201 using postman in my terminal

### DISCUSSION THREAD

**Discussion by devrainie** [DevSkillsHQ/discusssions#29] [Help] [answered]
Hi,

I am trying to past the last test case and when i test my backend api using postman i get: '"POST /presentations HTTP/1.1" 201 177'. I am aiming to get 201 on my test case. But when the test is run, it shows: '"POST /presentations HTTP/1.1" 200 26'. I am not sure what the issue is. I have tried changing http to https but https is not supported.

The very last post request is the one i did using postman. The one before is the test case.
<img width="1077" alt="test cases result" src="https://github.com/DevSkillsHQ/discusssions/assets/53815763/adcdeef2-b930-48bf-bdce-d2b23501aad9">

**Comment by fenske** [comment, marked as answer]
Hi, @rainxx3. I'm Anton from Alva. 

I changed the expected status to 200 just to see how much further the test would execute, and it failed on the following line where it checks for the `presentationId`, meaning it's not just the response code is wrong for some reason, the logic for creating a presentation didn't seem to be ever executed. 

My guess is that this request gets intercepted by either some default view or middleware that by default returns a 200.

If I were you, I'd run the app in the debug mode and examine the request differences from inside a debug process for both the Postman case vs. Cypress.

Hope this will lead you in the right direction.

I wish you the best of luck with the rest of your interview process!

BR,
Anton.
